src/main/resources/templates/about.html

```diff
@@ -0,0 +1,201 @@
+
+            <h1>Best Practices</h1>
+            <p>In order to ensure that your workflow is well presented in CWL Viewer, we recommend the following of
+            <a href="http://www.commonwl.org/user_guide/rec-practices/">CWL Best Practices</a>. Those which are
+                specifically relevant to the viewer are detailed below, but it is suggested that you try to meet as
+                many as possible to include the general quality and reproducibility of your workflows.</p>
+            <p>Some limitations of the CWL Viewer which you may need to be aware of are also described here.</p>
+
+            <h2>Label Strings</h2>
+            <p class="recommendation">Include a top level short <code>label</code> summarising each tool and workflow</p>
+            <p class="why">Labels give the user an easy human-readable version of the name for the tool or workflow</p>
+            <p class="use">For workflows this will be displayed at the top of the page as the title and for tools it will be
+                displayed in the table and as the name of the step in the visualisation. If a <code>label</code>
+                is given at the step level, it will take priority over the top level tool <code>label</code>. You can
+                use this to provide a more descriptive label of the tool's application in the particular step if
+                preferred.</p>
+
+            <h2>Doc Strings</h2>
+            <p class="recommendation">If useful, include a top level <code>doc</code> string providing a longer, more detailed description
+                than was provided in the <code>label</code> (see above)</p>
+            <p class="why">Docs give the user a detailed description of the role a tool or workflow performs</p>
+            <p class="use">For workflows this will be displayed at the top of the page under the title and for tools it will be
+                displayed in the table. If a <code>doc</code> string is given at the step level, it will take priority
+                over the top level tool doc. You can use this to provide a more descriptive label of the tool's
+                application in the particular step if preferred</p>
+
+            <h2>Conceptual Identifiers</h2>
+            <p class="recommendation">All <code>input</code> and <code>output</code> identifiers should reflect their conceptual identity.
+               Generic and uninformative names such as <code>result</code> or <code>input</code>/<code>output</code>
+               should be avoided</p>
+            <p class="why">Helpful identifiers allow for the links between steps in the CWL file to be easily distinguished</p>
+            <p class="use">Identifiers are displayed in the tables and are unique to the step. If <code>label</code> or
+               <code>doc</code> strings are for some reason not provided, the IDs will be used in the visualisation</p>
```

Comment:
This implies that the `label` or `doc` shows up in the graph -- I know you mean in the listing below, but that could be clarified.